### PR TITLE
Files: Android download improvements

### PR DESCRIPTION
**Pull request by songgao**
* fix download card style on mobile
* fix download bar style on mobile; make it horizontally scrollable on mobile
* auto dismiss completed download on Android after 3s, from left to right, one-by-one
* modernize `fs/footer/download*`!

Demo for auto-dismiss: /keybase/team/keybase/videos/android-download-20180305.mov

### CODE REVIEW

**Comment by songgao** [comment]
bump!

**Comment by songgao** [comment]
⪮

**Comment by songgao** [comment]
⪮

**Comment by songgao** [comment]
⪮

**Comment by songgao** [comment]
bump!

File: shared/fs/footer/download-wrapper.native.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// @flow
+import * as React from 'react'
+import Download, {type Props} from './download'
+import {NativeAnimated, NativeEasing} from '../../common-adapters/native-wrappers.native'
+
+type State = {
+  opacity: NativeAnimated.AnimatedValue,
+}
+
+export default class extends React.PureComponent<Props, State> {
```
Comment (chrisnojima) [inline shared/fs/footer/download-wrapper.native.js]: should name this class so it shows up in tooling better

File: shared/fs/footer/download-wrapper.native.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  }
+
+  _update = () => {
+    this.props.isFirst && this.props.isDone ? this._ensureStarted() : this._ensureStopped()
+  }
+
+  componentDidMount() {
+    this._update()
+  }
+  componentDidUpdate() {
```
Comment (chrisnojima) [inline shared/fs/footer/download-wrapper.native.js]: likely should stop on unmount

File: shared/fs/footer/download-wrapper.native.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+// @flow
+import * as React from 'react'
+import Download, {type Props} from './download'
+import {NativeAnimated, NativeEasing} from '../../common-adapters/native-wrappers.native'
```
Comment (chrisnojima) [inline shared/fs/footer/download-wrapper.native.js]: rarely import from native-wrappers just use `common-adapters/mobile.native`

**Comment by songgao** [comment]
@chrisnojima ready for another look!

**Pull request merged**